tests/mesh/mesh_assign.C

```diff
@@ -47,9 +47,9 @@ public:
   void testMeshMoveAssign()
   {
 
-    // We will do 4 tests:
-    // {ReplicatedMesh ,DistributedMesh} x {Mesh 2 from Memory, From File}
-    std::array<std::string, 2> mesh_types = {"replicated", "distributed"};
+    // We will do 6 tests:
+    // {Mesh, ReplicatedMesh ,DistributedMesh} x {Mesh 2 from Memory, From File}
```

Comment:
Oh, this was a smart addition.  I should have noticed the omission.  I foolishly think of Mesh as a typedef (what we really wanted) rather than a shim class (what we implemented, for backward compatibility with forward declarations)...